src/Components/OATHeader/internal/FormSettings.tsx

```diff
@@ -28,16 +29,33 @@ export const FormSettings = ({
 }: IModal) => {
     const { t } = useTranslation();
     const [namespace, setNamespace] = useState(state.namespace);
+    const [namespaceError, setNamespaceError] = useState(false);
     const headerStyles = getHeaderStyles();
 
     const handleProjectNamespaceChange = (value) => {
+        // Validate value contains only letters, digits, and underscores. The first character may not be a digit, and the last character may not be an underscore.
+        const regex = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
+        if (
+            (value.match(regex) && value.charAt(value.length - 1) !== '_') ||
+            value.length === 0
+        ) {
+            setNamespaceError(false);
+        } else {
+            setNamespaceError(true);
+        }
         setNamespace(value);
     };
 
     const handleOnSave = () => {
+        // If namespace is not provided revert to default value
+        let commitValue = namespace;
+        if (!namespace) {
+            commitValue = OATNamespaceDefaultValue;
```

Comment:
Can we not commit an empty string? Or null?

I.e. when we load the dialog the next time, we don't want to see the default value in the text field...